src/main/java/org/zalando/nakadi/service/FeatureToggleService.java

```diff
@@ -39,7 +39,8 @@ default void checkFeatureOn(final Feature feature) {
         KPI_COLLECTION("kpi_collection"),
         AUDIT_LOG_COLLECTION("audit_log_collection"),
         DISABLE_DB_WRITE_OPERATIONS("disable_db_write_operations"),
-        DISABLE_LOG_COMPACTION("disable_log_compaction");
+        DISABLE_LOG_COMPACTION("disable_log_compaction"),
+        FORCE_SUBSCRIPTION_AUTHZ("force_subscription_authz");
```

Comment:
Could we have a name that's more like the one in https://github.com/zalando/nakadi/pull/1068 ? We should try to be consistent whenever possible.